__tests__/wait.test.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Unit tests for src/wait.ts
+ * Unit tests for src/wait.js
```

Comment:
suggestion: I think we should also update line 7 (just outside the PR diff)

```
describe('wait.ts', () => {
```

Reply:
Thanks for catching that one!

Reply:
Updated!